ExampleApp/ExampleViewController.swift

```diff
@@ -471,6 +471,9 @@ class ExampleViewController: UIViewController, UITableViewDataSource, UITableVie
 
         for linkedLogin in linkedLogins {
```

Comment:
Is there a reason why we are not using the linkedBrokerManager.unlinkBroker method here ?